payments/credit-card/seamless-view.md

```diff
@@ -141,7 +352,7 @@ sequenceDiagram
     Payer ->>Payer: Initiate iFrame
     Payer->>+Merchant: access merchant page
     deactivate Payer
-    Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<paymentorder.id>
+    Merchant->>+SwedbankPay: GET /psp/creditcard/payments/<payment.id>
```

Comment:
Let's not use `<payment.id>` in a path like this, as the Payment ID is a full URL and we don't want to encourage URL building in the client.

```suggestion
    Merchant->>+SwedbankPay: GET <payment.id>
```